docs/setup_help/m1_mac_setup.md

```diff
@@ -0,0 +1,47 @@
+#Notes from Wes Goar on setup of vrs-python on a m1 mac
+- @author Wes Goar
+- date: 20220328
+- macOS version: 12.3
+- python version: 3.9.11
+
+##Notes
+- I installed seqrepo before trying this setup.
+- I had already installed xcode and homebrew on my machine.
+
+###Steps
+1. brew install openssl
+2. add the following statement in your .zshrc: export PATH="/opt/homebrew/opt/openssl@1.1/bin:$PATH"
+3. add the following statements in your .zshenv:
+4. Run brew info openssl to ascertain the correct environment path to put into the export statements
+
+export LDFLAGS="-L/opt/homebrew/opt/openssl@1.1/lib" \
+export CPPFLAGS="-I/opt/homebrew/opt/openssl@3/include" \
+export PKG_CONFIG_PATH="/opt/homebrew/opt/openssl@1.1/lib/pkgconfig"
```

Comment:
Two thoughts

1) Just curious -- do you have both openSSL v1.1 and v3 installed? I just have v3 (3.0.2 specifically) but, y'know, no idea when that got there (and I'm on an intel mac anyway).

2) Good practice for readability to put code like this into code blocks, eg

```shell
export LDFLAGS="-L/opt/homebrew/opt/openssl@1.1/lib" 
export CPPFLAGS="-I/opt/homebrew/opt/openssl@3/include" 
export PKG_CONFIG_PATH="/opt/homebrew/opt/openssl@1.1/lib/pkgconfig"
```

See https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks (I can never remember the syntax keyword for highlighting shell code, I think it's just `shell`)

Also, I don't think you need the trailing backslashes if you're just putting this into a shell profile file

Reply:
1. Yes I have both installed.

Reply:
2. Updated the exports to include syntax and be in code blocks.